ondemand/install.sh

```diff
@@ -99,3 +99,11 @@ log_info "Generating new httpd24 and dex configs.."
 
 yum clean all
 rm -rf /var/cache/yum
+
+log_info "Cloning repos to assist with app development.."
+mkdir -p /var/git
+git clone https://github.com/OSC/bc_example_jupyter.git --bare /var/git/bc_example_jupyter
+
+log_info "Enabling app development for hpcadmin..."
+mkdir -p /var/www/ood/apps/dev/hpcadmin
+ln -s /home/hpcadmin/ondemand/dev /var/www/ood/apps/dev/hpcadmin/gateway
```

Comment:
I think we also need a `su hpcadmin -c "mkdir -p /home/hpcadmin/ondemand/dev"` right?

Reply:
The dashboard creates this automatically if it doesn't exist. I was able to login as the hpcadmin user and access the sandbox apps and it worked.

Reply:
Just did a quick test and looks like it's created ok:

```[hpcadmin@ondemand ~]$ ls -alh /home/hpcadmin/ondemand/dev
total 8.0K
drwxr-xr-x 2 hpcadmin hpcadmin 4.0K Jul 15 19:46 .
drwxr-xr-x 4 hpcadmin hpcadmin 4.0K Jul 15 19:46 ..
```